pkg/commands/app.go

```diff
@@ -771,6 +780,53 @@ func NewPluginCommand() *cli.Command {
 	}
 }
 
+// NewK8sCommand is the factory method to add k8s subcommand
+func NewK8sCommand() *cli.Command {
+	return &cli.Command{
+		Name:    "kubernetes",
+		Aliases: []string{"k8s"},
+		Usage:   "scan kubernetes images and misconfigurations",
```

Comment:
I think "scan kubernetes images and configurations" or "scan kubernetes vulnerabilities and misconfigurations" are better since `image` is an artifact and `misconfigurations` is a security issue.